sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -133,7 +133,10 @@ class PlanResolutionSuite extends AnalysisTest {
       analyzer.ResolveRelations,
       new ResolveCatalogs(catalogManager),
       new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v")),
-      analyzer.ResolveTables)
+      analyzer.ResolveTables,
+      analyzer.ResolveReferences,
```

Comment:
nit: Should these be in the same order as they are in the actual analyzer? If so, how do we make sure that we dont break this constraint? At the very least this should be commented as it is critical for the tests to run correctly .. isnt it?

This is probably not relevant for this PR, but I am curious.... instead of pulling out a few rules from the analyzer, why are we not using the Analyzer directly (with all the rules) to parse and resolve?

Reply:
I tried to use the analyzer then a lof of tests failed. I've added a TODO to fix it.